views/js/qtiCreator/editor/widgetToolbar.js

```diff
@@ -383,11 +384,11 @@ define([
                 buttonConfig = handleConfig(buttonConfig, barConfig.extendDefaultConfig);
                 barConfig = $.extend(defaultBarConfig, (barConfig || {}));
 
-
                 toolbar = buildToolbar(buttonConfig, target, tlbWrapper);
 
-                if(barConfig.title){
-                    title = $('<span>', {text : barConfig.title, 'class' : 'tlb-title', title : barConfig.title});
+                if (barConfig.title) {
+                    var getQtiClassTitle = qtiClass => classTitles[qtiClass] || qtiClass;
```

Comment:
Maybe better to move this in `views/js/qtiCreator/helper/classTitles.js`?
Because it is repeated in `views/js/qtiCreator/widgets/interactions/Widget.js`

Reply:
Will be nice, working on

Reply:
done, thanks